libphx/script/ffi/LineSegment.lua

```diff
@@ -5,10 +5,10 @@ local LineSegment
 
 do -- C Definitions
     ffi.cdef [[
-    void LineSegment_ToRay    (LineSegment const*, Ray*);
-    void LineSegment_FromRay  (Ray const*, LineSegment*);
-    cstr LineSegment_ToString (LineSegment*);
-  ]]
+        void LineSegment_ToRay    (LineSegment*, Ray* out);
```

Comment:
Can we derive constness from the fact that `LineSegment*` is a `&self` in the Rust code (rather than a `&mut self`)?

Reply:
I was thinking about that. Do we need to do it only for **&self**/**&mut self** or for other parameters too? I.e. **data: &Data**/**data: &mut Data**.

Reply:
I think it applies for all parameters, like for example:
```
Matrix* Matrix_Clone(const Matrix* other);
```

Reply:
Ok. Will do it

Reply:
Done. Now we will see more const in the generated Lua wrappers than we have now

Reply:
Nice, I think that's a good thing.